### PR TITLE
Import/Export updates for dependent select list

### PR DESCRIPTION
**Pull request by nolanpro**
Requires the following PRs
- https://github.com/ProcessMaker/screen-builder/pull/1341
- https://github.com/ProcessMaker/package-collections/pull/335
- https://github.com/ProcessMaker/package-webentry/pull/186
- https://github.com/ProcessMaker/vue-form-elements/pull/381

## Issue & Reproduction Steps
Import/Export modifications needed for collections select list

## Solution
- Modify screen exporter

## How to Test
- Export a process with a screen that has a collection select list

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7437

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:screen-builder:feature/FOUR-7437
ci:vue-form-elements:feature/FOUR-7437
ci:package-collections:feature/FOUR-7437
ci:package-webentry:feature/FOUR-7437
......

### CODE REVIEW

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed Could not open input file: deploy-stm/get-url.php

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

**Comment by pmkerberos** [comment]
QA server was successfully deployed https://ci-ddab00c6f8.eng.processmaker.net

File: tests/Feature/ImportExport/UtilsTest.php [hunk 1]
```diff
@@ -31,4 +32,61 @@ public function testGetPmConfig()
         $config = Utils::getPmConfig($result->first());
         $this->assertEquals('list', $config['endpoint']);
     }
+
+    public function testFindScreenDependent()
+    {
+        $config = [
```
Comment (julceslauhub) [inline tests/Feature/ImportExport/UtilsTest.php:38]: maybe this item can be a json in the "fixtures" folder

**Review by julceslauhub — COMMENTED** [review]
Added a suggestion

File: tests/Feature/ImportExport/UtilsTest.php (hunk 1 above)
Comment (nolanpro) [reply tests/Feature/ImportExport/UtilsTest.php:38]: Good idea, fixed

**Pull request merged**